surf.deckr.deckr.metainfo.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- 
+    Generator for metainfo & .desktop files:
+https://www.freedesktop.org/software/appstream/metainfocreator/#/
+-->
+<component type="desktop-application">
+  <id>surf.deckr.deckr</id>
+  <name>Deckr</name>
+  <summary>Deckr Desktop App</summary>
+  <developer_name>Steve Gehrman</developer_name>
+  <url type="homepage">https://deckr.surf</url>
+  <metadata_license>FSFAP</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
+  <supports>
+    <control>pointing</control>
+    <control>keyboard</control>
+  </supports>
+  <description>
+    <p>Desktop app used with the Deckr chrome plugin. Deckr is a bookmark manager which allows you to share and download bookmark "decks" with others online.</p>
+    <p>Features</p>
+    <ul>
+      <li>Replaces "New Tab" in chrome with a bookmark deck manager</li>
+      <li>Share and download bookmark decks online</li>
+      <li>Lots more features. Install it and give it a try.</li>
+    </ul>
+  </description>
+  <provides>com.cocoatech.deckr</provides>
+  <replaces>com.cocoatech.deckr</replaces>
```

Comment:
```suggestion
  <provides>
    <id>com.cocoatech.deckr</id>
  </provides>
  <replaces>
    <id>com.cocoatech.deckr</id>
  </replaces>
```

Reply:
This is shown in the docs https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#replaces